xunit-autolabeler-v2/ast_parser/python/source_parsers/webapp2_router.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, List
+
+
+def parse(nodes: List[Any]) -> List[Any]:
+    """Identify webapp2 route-handling snippets in Python
```

Comment:
Nit/question: worth linking to the design doc here? @tmatsuo @martinvarelaj

Reply:
Maybe

Reply:
If the DD is publicly available, sure I don't see any reason why we shouldn't link to it.